### DISCUSSION TITLE
Greetings from a previous react-mirror implementation

### DISCUSSION THREAD

**Discussion by Theadd** [iamogbz/react-mirror#1413] [Show and tell]
Hi @iamogbz !

I just stumbled upon this project by pure chance and I was really surprised because I had a lot of fun implementing exactly the same project a while ago, even with the same name! =) But with the stack from 9 years ago (React v0.14 w/ ES6), just before react-portal came out, being completely obsolete for years now. Nice to see yours in **TS** + React **v18.0** + hooks.

<blockquote>

![react-mirror](https://github.com/user-attachments/assets/e2c97e3b-e8df-4c01-b1c3-4292270ec756)
<sub>*<u>Image Description</u>: Multiple fully synchronized and eventful __exact__ mirrors of a single game board `surface`. All* </sub>

</blockquote>

<sup>[[LIVE DEMO / PLAYGROUND](https://theadd.github.io/react-mirror/)]</sup>

[Here](https://theadd.github.io/react-mirror/demos/react-motion-demo2/)'s another demo, displaying a seamless mirror of an animated [`react-motion`](https://github.com/chenglou/react-motion)-based <em>"physics-"</em>game to prove its full potential at reflecting the most complex cases.

I achieved the *"eventful"* part just by setting the `pointer-events` css rule to `all` on the replica currently being hovered and `none` to all other replicas (mirrors).
[![image](https://github.com/user-attachments/assets/e11b574d-da10-4707-aee0-bdc9a5a7b9ab)](https://github.com/Theadd/react-mirror/blob/9c2fcb02c5f2c28f24cc526ec3f25add28c6d7e1/src/mirror.js#L44-L46)

Well, although implementing it was quite fascinating, unfortunately, I did not see many practical applications for it.

It would be great if you could add a better demo. I've [added your repo](https://github.com/Theadd/react-mirror/tree/master?tab=readme-ov-file#react-mirror) as an up to date alternative to mine.

**Comment by iamogbz** [comment]
@Theadd that's pretty cool to see thanks! I'm open to including a more complex demo, if you have some easily shareable html element/component that would be a good example send me a link.
